demo-forms/custom_contacts/README.MD

```diff
@@ -0,0 +1,11 @@
+# Applying Sample Custom Contact Types
```

Comment:
Let's add a link with a brief "oh yeah - custom contact types!" sentence to this document on the [current readme](https://github.com/medic/cht-core/blob/custom_contacts/demo-forms/readme.md).  This will facilitate discovery among other thing.